docs/user-guide/title-section.md

```diff
@@ -71,4 +71,16 @@ In this case the background editor toolbar allow the user to customize the image
 * **Change field theme** <img src="../img/button/change-field-theme.jpg" class="ms-docbutton"/> to set the colour of the empty background between *Bright* and *Dark*
 
 !!! warning
-    The *Align content* and the *Change field theme* buttons are disabled if the image size is full screen.
\ No newline at end of file
+    The *Align content* and the *Change field theme* buttons are disabled if the image size is full screen
+
+### Maps
+
+In this case, adding a map as background, the result will be like this:
+
+<img src="../img/title-section/map-background.jpg" class="ms-docimage"/>
+
+The background toolbar, in case of the map, has a new button: 
```

Comment:
Specific that is new compared to the images one. Compared to the images one one button is also missing